test/WebJobs.Script.Tests/FileMonitoringServiceTests.cs

```diff
@@ -19,5 +26,66 @@ public static void GetRelativeDirectory_ReturnsExpectedDirectoryName(string path
         {
             Assert.Equal(expected, FileMonitoringService.GetRelativeDirectory(path, @"C:\Functions\Scripts"));
         }
+
+        [Theory]
+        [InlineData("app_offline.htm", 150, true, false)]
+        [InlineData("app_offline.htm", 10, true, false)]
+        [InlineData("host.json", 0, false, false)]
+        [InlineData("host.json", 200, false, false)]
+        [InlineData("host.json", 1000, false, true)]
+        public static async Task TestAppOfflineDebounceTime(string fileName, int delayInMs, bool shutdownCalled, bool restartCalled)
```

Comment:
nit: rename `shutdownCalled` and `restartCalled` to `expectShutdown` and `expectRestart`